src/components/ui/Select/select.scss

```diff
@@ -1,6 +1,6 @@
 @import "../../../styles/scaffold.scss";
 
-.sort {
+[data-ui-select] {
```

Comment:
A new `data-ui-*` pattern emerges... 👁️

Reply:
Oops, that was not my intention. I was just using the component name and was also fearing that it could conflict with some existing `data-select`. But just checked on `faststore` that it is `data-store-select`. 

I'll change it to just `data-select`.